public/index.html

```diff
@@ -25,6 +25,14 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
```

Comment:
Aqui ta certo linkar o bootstrap mesmo?? Eu to meio por fora, mas não consegui achar nada que usasse o bootsrap nos arquivos que mudaram. O Material usa bootstrap??